notificationworkerlambda/src/main/scala/com/gu/notifications/worker/models/SendingResults.scala

```diff
@@ -84,5 +84,14 @@ object LatencyMetrics {
       }
     }.getOrElse(previous) // If the overall batch was a failure (or the batch only contained failures) just return the previous result
   }
+
+  def audienceSizeBucket(audienceSize: Option[Int]): String = audienceSize match {
+    case None => "unknown"
+    case Some(audience) if audience < 100000 => "S"
+    case Some(audience) if audience < 500000 => "M"
+    case Some(audience) if audience < 1000000 => "L"
+    case _ => "XL"
+  }
+
```

Comment:
Bucketing has been set initially as:
1 - 99,999 (captures most tag specific notifications)
100,000 - 499,999(captures au or us breaking news notifications)
500,000 - 999,999 (captures uk breaking news notifications)
1,000,000 + (captures combined region breaking notifications)

This was determined by looking at the [ophan-app-notifications page](https://dashboard.ophan.co.uk/notifications)

Numbers can be easily modified if we need to or new dimensions can be added if we need to add more distinctions. Be aware though that each new bucket we add will be adding 5 new dimensions (one each for "android, android-beta, android edition, ios and ios-edition) so do this with care.

Reply:
> * 1 - 99,999 (most tag specific notifications);

This should also capture breaking sport notifications.